Library/Formula/arpwatch.rb

```diff
@@ -0,0 +1,46 @@
+class Arpwatch < Formula
+  desc "tool for monitoring ARP traffic"
+  homepage "http://ee.lbl.gov/"
+  url "ftp://ftp.ee.lbl.gov/arpwatch-2.1a15.tar.gz"
+  sha256 "c1df9737e208a96a61fa92ddad83f4b4d9be66f8992f3c917e9edf4b05ff5898"
+
+  # needs libpcap
+  depends_on "libpcap"
+
+  def install
+    # patch makefile to avoid permission errors
+    chmod 0644, %w[Makefile.in]
+    inreplace "Makefile.in",
+      "$(INSTALL) -m 555 -o bin -g bin arpwatch $(DESTDIR)$(BINDEST)",
+      "mkdir -p $(DESTDIR)$(BINDEST)\n\t$(INSTALL) -m 555 arpwatch $(DESTDIR)$(BINDEST)"
+    inreplace "Makefile.in",
+      "$(INSTALL) -m 555 -o bin -g bin arpsnmp $(DESTDIR)$(BINDEST)",
+      "$(INSTALL) -m 555 arpsnmp $(DESTDIR)$(BINDEST)"
+    inreplace "Makefile.in",
+      "$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpwatch.8 \\",
+      "mkdir -p $(DESTDIR)$(MANDEST)\n\t$(INSTALL) -m 444 $(srcdir)/arpwatch.8 \\"
+    inreplace "Makefile.in",
+      "$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpsnmp.8 \\",
+      "$(INSTALL) -m 444 $(srcdir)/arpsnmp.8 \\"
+
+    # patch wrong exitcode when showing usage, so test works
+    chmod 0644, %w[arpwatch.c]
+    inreplace "arpwatch.c",
+      "\" \[-n net\[\/width\]\] \[-r file\]\\n\", prog\);\n\texit\(1\);",
+      "\" \[-n net\[\/width\]\] \[-r file\]\\n\", prog\);\n\texit\(0\);"
```

Comment:
I poked around the web a bit on this. It looks like arpwatch is widely available in Linux distros and used a lot, but is one of those old school BSD utilities that is effectively abandonware by Homebrew's standards. I think we'd reject this for the Homebrew core too.

But there's some contact info in the man page.

```
.\" @(#) $Id: arpsnmp.8,v 1.5 2000/09/17 20:34:41 leres Exp $ (LBL)
...
AUTHORS
       Craig  Leres  of	 the Lawrence Berkeley National Laboratory Network Re-
       search Group, University	of California, Berkeley, CA.

       The current version is available via anonymous ftp:

	      ftp://ftp.ee.lbl.gov/arpwatch.tar.gz

BUGS
       Please send bug reports to arpwatch@ee.lbl.gov.
```

You could submit the patches there and see what happens. Given that that's a 15-year-old document, dunno how much luck you'll have.

It looks like `arpwatch` is included in FreeBSD and actively maintained [as a port there](https://svnweb.freebsd.org/ports/head/net-mgmt/arpwatch/). Maybe we could treat FreeBSD as the upstream in this case, like [rpm2cpio](https://github.com/Homebrew/homebrew/blob/c31dc2042f2e3f615e6f7c965b25821e4f633e98/Library/Formula/rpm2cpio.rb) and [sntop](https://github.com/Homebrew/homebrew/blob/8c6777b2b8e847d5f2b447eae532fa8ecffa6467/Library/Formula/sntop.rb) do. Would they accept a patch like this?